### PR TITLE
[Admin] Added collapse menu

### PR DESCRIPTION
**Pull request by TefDesign**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

Add collapse on admin sidebar 

sylius_admin_menu

### CODE REVIEW

**Comment by lchrusciel** [comment]
However, can you rebase to the newest master and check how will it behave together with #11571?

**Comment by lchrusciel** [comment]
Also, will it remember which cards have been collapsed and which not? It could be annoying for users to always toggle them.

**Comment by TefDesign** [comment]
Hi @lchrusciel 
Thanks :)
For #11571 , it's already present in my pull request :smiley:  : 
image

I created my pull request from the master so the task is there :)
For your question, I am using the "current_ancestor" class already present and I have not touched the html structure. With this class, if an item is active, the entire block remains open.
In the JavaScript function, I have provided a parameter that allows opening a menu to close the others if a user prefers this system. All JavaScript script is documented in JSDoc standards !

**Comment by TefDesign** [comment]
@lchrusciel I will still check something and I tell you :)

**Comment by TefDesign** [comment]
@lchrusciel Rebase ok and check ok :+1:  : 
image

**Comment by lchrusciel** [comment]
Thanks, Stéphane! :tada:

File: src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss [hunk 1]
```diff
@@ -97,6 +96,51 @@ a {
         background: $sylius-brand-color !important;
         border-radius: 0 99px 99px 0 !important;
     }
+    .item {
+        padding-bottom: 5px;
+        .menu {
+            visibility: hidden;
+            overflow: hidden;
+            max-height: 0;
+            transition: all .5s ease-in-out;
```
Comment (vasilvestre) [inline src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss:105]: https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108
https://stackoverflow.com/a/12347702/6796253
These links seem to fix the actual problem. :)

**Pull request merged**